SYCL/ESIMD/api/functional/operators/operator_decrement_and_increment.hpp

```diff
@@ -0,0 +1,254 @@
+//===-- operator_decrement_and_increment.hpp - Functions for tests on simd
+//      increment and decrement operators definition. ---------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file provides functions for tests on simd decrement and increment
+/// operators.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "common.hpp"
+#include "../mutator.hpp"
+
+namespace esimd = sycl::ext::intel::experimental::esimd;
+namespace esimd_functional = esimd_test::api::functional;
+
+namespace esimd_test::api::functional::operators {
+
+// Descriptor class for the case of calling constructor in initializer context.
+struct pre_decrement {
+  static std::string get_operator_type() { return " pre decrement"; }
```

Comment:
```suggestion
  static std::string get_operator_type() { return "pre decrement"; }
```
Looks like unintended whitespace left for all 4 test cases

Reply:
this comment was applied [here](https://github.com/intel/llvm-test-suite/pull/827/commits/07503714b6bf2d29f8318597e51142bec03a1fb8#diff-957362298136f7a2e0ab66c1cff4239caca52ea84dab6e6b2728ccc9411ebe0eL28)